example.jsonnet

```diff
@@ -1,12 +1,18 @@
 local t = import 'kube-thanos/thanos.libsonnet';
+local minio = (import 'github.com/observatorium/observatorium/configuration/components/minio.libsonnet')({
```

Comment:
Hmm. Maybe we can define `minio.libsonnet` here? 
Let's not create a cyclic import. Currently, `observatorium/observatorium` also relies on kube-thanos as a dependency.

Reply:
Sure defining `minio.libsonnet` here sounds good